.github/workflows/create-draft-release-reusable.yml

```diff
@@ -0,0 +1,44 @@
+name: Create draft release (reusable)
+
+on:
+  workflow_call:
+    inputs:
+      VERSION:
+        required: true
+        type: string
+        description: The semantic version number.
+    secrets:
+      BOT_PAT:
+        required: true
+        description: The github personal access token of your bot.
+      GH_TOKEN:
+        required: true
+
+jobs:
+  create-draft-release:
+    name: Create a draft release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Write changelog to file
+        env:
+          VERSION: ${{ inputs.VERSION }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        shell: bash
+        run: |
+          # Note: your repository needs to have this script.
+          # Running this script should result in a file named CHANGELOG.md, located in the base directory.
+          ./hack/scripts/create_changelog.sh "${VERSION}"
+
+      - name: Print out changelog
+        run: cat CHANGELOG.md
```

Comment:
If there are no changes, what will be printed?

Reply:
That depends on the script. With what we have it will be only headlines and a link to the full changelogs.

Reply:
Does make sense here to print a message before and after the `cat` indicating the beginning and end of the changelog?

Reply:
That's the reason why I placed this in its own step. I makes it easy to find and easy to read.

```
|> Run cat CHANGELOG.md
Changelog blah blah blah
Changelog blah blah blah
Changelog blah blah blah
```

So to answer the question, no, I think it is good already.